docs/core/diagnostics/dotnet-trace.md

```diff
@@ -98,6 +98,16 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
 - **`--clreventlevel <clreventlevel>`**
 
   Verbosity of CLR events to be emitted.
+  The table belows show the list of avaiable event levels:
+  
+  | String Value | Numberic Value |
+  | ------------ | ------------------- |
+  | `logalways` | `0` |
+  | `critical` | `1` |
+  | `error` | `2` |
+  | `warning` | `3` |
+  | `informational` | `4` |
+  | `verbose` | `5` |
```

Comment:
Just a nit:
Usually the good practice I observe in this code base (and I believe: in general), is to justify the raw text (the table implementation) the way it should look like when actually displaying. Such formatting ease the further development, and improves the readability.
So, I would suggest something like below:
```suggestion
  | String Value    | Numeric Value |
  | --------------- | :-----------: |
  | `logalways`     |      `0`      |
  | `critical`      |      `1`      |
  | `error`         |      `2`      |
  | `warning`       |      `3`      |
  | `informational` |      `4`      |
  | `verbose`       |      `5`      |
```
Note I also justify the `Numeric value` column as center, and fixed the `Numberic` typo.